src/strings/csv/es.csv

```diff
@@ -669,6 +669,7 @@ PREFERRED_WEIGHT_SYSTEM,Sistema de peso preferido
 PREVIOUS_ARROW,< Anterior
 PRIVACY_POLICY,Política de privacidad
 PROBLEMS,Problemas
+PROBLEMS_WITH_ENTRIES,Se produjo un problema con algunas entradas de este formulario.
```

Comment:
@constanzauanini does this look fine for `There was a problem with some entries on this form.`?